nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
Oh, it should be in `specialArgs` apparently. I didn't get the meaning from the description (and I still don't).